google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -434,7 +434,7 @@ interfaces:
     max_retry_delay_millis: 60000 # 60 seconds
     initial_rpc_timeout_millis: 12000 # 12 seconds
     rpc_timeout_multiplier: 1
-    max_rpc_timeout_millis: 12000 # 12 seconds
+    max_rpc_timeout_millis: 30000 # 30 seconds
```

Comment:
Gotcha. Apologies for my confusion. (I realized that last week, but forgot about it when I looked at this today.)